python/py-coolprop/Portfile

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        CoolProp CoolProp 6f4dfa82c1232f4827a81040284262a7099ed3fe
+revision            1
+fetch.type          git
+
+checksums           rmd160  e09935be3be632239bf631e5bea3e54fd68f39e1 \
+                    sha256  fd26827cdb70ffcef657538a2e1105d6a46beda6d898c385b96961e5b46e15e8 \
+                    size    8002925
+
+
+name                py-coolprop
+categories-append   science
+license             MIT
+maintainers         openmaintainer @jsalort
+platforms           darwin
```

Comment:
this is the default value for the `platforms` field since MacPorts v2.7 so the line can be removed